lib/TemplatedPathPlugin.js

```diff
@@ -139,6 +146,7 @@ const replacePathVariables = (path, data) => {
 			)
 			.replace(REGEXP_ID, getReplacer(chunkId))
 			.replace(REGEXP_MODULEID, getReplacer(prepareId(moduleId)))
+			.replace(REGEXP_MODULE_EXT, getReplacer(moduleExtension))
 			.replace(REGEXP_NAME, getReplacer(chunkName))
 			.replace(REGEXP_FILE, getReplacer(data.filename))
 			.replace(REGEXP_FILEBASE, getReplacer(data.basename))
```

Comment:
basename doesn't need to be passed here but could be extracted from `data.filename`.